bloc-did-method.md

```diff
@@ -8,99 +8,165 @@ The `did:bloc` DID method allows groups of independent organizations to share cu
+TODO: this should probably be in a secondary file, or stored in a way that's easy to detach without risk of screwing up the payload text
+- A detached JWS signature on the file, created by the stakeholder
 
 ##### History
 The `history/` directory contains historical consortium configs. Each such file is named `[hash].json`, where `[hash]` is the SHA-256 hash of the given file.
 
+TODO: instead of separate history files, have a complete history database file, as either an object or a concat JSON?
+
+TODO: consortium history may need to cache the stakeholder configs and verkeys - since these can be replaced on the stakeholder domain (or the stakeholder could disappear completely), with the associated consortium configs being subsequently unverifiable.
+
+
+##### Stakeholder Files
+    .well-known/did-bloc/
+        [domain].json
+        history/
+
+`[domain].json` is the stakeholder configuration file for this stakeholder.
+
+`history/` is a directory of previous stakeholder configs for this stakeholder
+
+TODO: is a history for the stakeholder config necessary? If the consortium caches historical stakeholder info - specifically, DID -> verkey mappings so historical consortium files can be verified, then the stakeholders don't need to keep historical data, or even exist any longer.
+
 <!--
 TODO:
 - Resolver has to check for consortium changes - how often?
   - update check frequency or cache lifetime should be part of the policy, or specified in the particular configuration file
 - when the resolver hits stakeholders to verify the consortium policy config, if it finds that a stakeholder has a newer version of a file than the consortium, how does it respond?
  - update to the latest version that has sufficient endorsement?
+ 
+- Policy may specify that a number of independent queries have to be made (queries made to endpoints 
+  from different organizations in the consortium). TODO is to justify why.
+  - This isn't about independent queries for regular use, but independent queries when bootstrapping trust, to verify that stakeholders named in the consortium config have signed the consortium config (and signed their own stakeholder config) with keys that are associated with did docs on the ledger
+  - maybe we can suggest resolving stakeholder X's did doc using stakeholder Y's endpoint, to verify the stakeholders are on the same ledger?
+    - but then, if they all sign the same consortium config, which points to all of them, we don't need this complication
 --> 
+### Bootstrapping Trust
+
+The Bloc DID method is designed to offer several mechanisms for a resolver to confirm that it can trust the stakeholders to provide the DID doc ledger service. 
 
-### CRUD Operation Definitions
+In each mechanism, the resolver has the option to cache verified configuration files for a period of time (the cache lifetime being specified in the configuration), and in some of these, the resolver can automatically fetch and verify updates.
```

Comment:
Or in the [`Cache-Control`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) header

Reply:
Yes